examples/k8s/cluster-role.yaml

```diff
@@ -22,6 +22,10 @@ rules:
   - get
   - list
   - watch
+  - create
+  - update
+  - patch
+  - delete
```

Comment:
This looks over-done: I think you want extra permissions just on `nodes` so pull that out to a separate rule.
I don't see why `create` or `delete` would be needed by Scope.